Tnw homepage

Blaise-Shyaka opened the pull request:
In this project, we built a clone of the next web magazine homepage. We have:

- Written the markup for the page
- Started styling the page using a mobile-first approach
- Made the navigation bar sticky to the top
- Written media queries targeting 768px and 1024px screen widths
- Used a grid to layout sections

elasfarc committed:
add .DS_Store to .gitignore

elasfarc committed:
init html & Css

elasfarc committed:
structure Nav, Hero, latest-funding HTML sections

elasfarc committed:
complete HTML structure

structure card-container-grid class, latest-deals and footer

Blaise-Shyaka committed:
correctly align elements in the footer

elasfarc committed:
add fonts dir

./assets/fonts

Blaise-Shyaka committed:
add text and images in the hero and latest-news sections

Blaise-Shyaka committed:
Merge branch 'TNW-homepage' of https://github.com/elasfarc/TheNextWeb-responsive-clone into TNW-homepage

Blaise-Shyaka committed:
add images and markup to categories class cards

elasfarc committed:
add content to .latest-deals-card

elasfarc committed:
adding imgs to .catagory-card

elasfarc committed:
CSS reset & base styles

css first
CSS reset / base styles / VARs

elasfarc committed:
h1 default font properties

elasfarc committed:
style .hero

setting the hero grid container
setting general hero > section properties
setting background for each section

elasfarc committed:
.hero section structure modification

wrap h1,p,div inside .hero > section into a div to position the whole div to the bottom

elasfarc committed:
.hero tablet style

modify the grid container to 2r*3c

elasfarc committed:
select the apropriate selectors

for .hero section .text-wrap

elasfarc committed:
Update styles.css

Blaise-Shyaka committed:
fix w3c validator errors

Blaise-Shyaka committed:
install linters and fix stylelint errors

elasfarc committed:
adding orange-heading class

to style main sections heading

elasfarc committed:
style latest-news class for mobile

elasfarc committed:
style latest news class for iPad & add fonts

- change
                .latest-news article h2 -> font to 'avalon-reg'

elasfarc committed:
style latest-news class for Desktop

elasfarc committed:
latest-news mobile no. of Grid rows

change the no. from explicitly (8) to (auto)

Blaise-Shyaka committed:
finish styling .card-container-top

elasfarc committed:
style last-funding mobile

elasfarc committed:
style latest-funds iPad

elasfarc committed:
latest funds ul li

Blaise-Shyaka committed:
add comments and fix stylelint errors

elasfarc committed:
nav -> sticky-nav styling

Blaise-Shyaka committed:
style latest deals and change the name of content-wrap

Blaise-Shyaka committed:
rename conflicting class names

elasfarc committed:
sticky-nav sub p

elasfarc committed:
conflict fixing

elasfarc committed:
Merge branch 'TNW-homepage' of https://github.com/elasfarc/TheNextWeb-responsive-clone into TNW-homepage

elasfarc committed:
styling NAV -> yellow sticky & sticky-top-nav /MOBILE/

elasfarc committed:
TNW nav logo hover effect

elasfarc committed:
top black nav content & hide in mobile

elasfarc committed:
add links to sections -> top-nav

elasfarc committed:
resturcture the NAV

split the nav content into two NAVs and one div
black, white and yellow

elasfarc committed:
style black nav

elasfarc committed:
style white nav mobile

elasfarc committed:
style .yellow div for MOBILE

elasfarc committed:
.white left logo mobile

elasfarc committed:
style black and white nav for IPAD

elasfarc committed:
style .white .section for iPad

elasfarc committed:
style  top yellow div iPad

elasfarc committed:
style nav for desktop

elasfarc committed:
fixing black nav overflow at iPad breakpoint

elasfarc committed:
nav icons pointer

elasfarc committed:
last-funding last card

wrap .vlaue by parent .second-row (the last card)

elasfarc committed:
fix styling for last-funding heading

elasfarc committed:
last funding cards font

elasfarc committed:
.hero section:first-child desktop

Blaise-Shyaka committed:
style the footer

Blaise-Shyaka committed:
fix stylelint errors

Blaise-Shyaka committed:
style the hero in desktop view

elasfarc committed:
conflicting fix

Blaise-Shyaka committed:
fix merge conflicts

Blaise-Shyaka committed:
fix stylelint errors

Blaise-Shyaka committed:
fix latest-funding width

Blaise-Shyaka committed:
fix latest-funding width

Blaise-Shyaka committed:
edit the README

Blaise-Shyaka committed:
fix html validator errors and warnings

Blaise-Shyaka committed:
delete commented out code

elasfarc committed:
stylelint errors fix

Blaise-Shyaka committed:
fix stylelint errors and correctly format HTML

Blaise-Shyaka committed:
fix linting errors

lmaldonadoch commented on assets/fonts/aside/transfonter.org-20200813-182021/demo.html:
```diff
@@ -0,0 +1,207 @@
+<body>
+<div class="page">
+    <div class="demo">
+        <h1 style="font-family: 'Avalon'; font-weight: normal; font-style: normal;">Avalon</h1>
+        <pre>.your-style {
+    font-family: 'Avalon';
+    font-weight: normal;
+    font-style: normal;
```
- [ ] [REQUIRED] Inline style is a horrible, horrible practice. Please move all styling to a CSS file.

Blaise-Shyaka committed:
fix webhint error

Blaise-Shyaka committed:
remove unusual character from the html

lmaldonadoch reviewed (changes requested):
# [STATUS] Changes Required 🥇 🏃‍♂️

## Great job, team!!! 🏆

This was me reviewing your project:

<img src="https://media.giphy.com/media/MuTenSRsJ7TQQ/giphy.gif" />

## It is **A-MA-ZING**

The quality of the work is impressive, your project looks so similar to the original website, that I confused more than once your work with the real deal. **Breathtaking!**

There are some things that we need to take care of before moving on to the next project, but with your talent, I'm sure they will be ready in no time!

## Large screen

- [x] [REQUIRED] Please fix these links wrapping down, and the margin between elements. (This also happens on medium screens)

![image](https://user-images.githubusercontent.com/59229758/90456798-78e6fa00-e0bf-11ea-8e82-41fbd6e5e0ee.png)

- [x] [REQUIRED] You are missing the blue logo at the bottom of the page. Please add it.

![image](https://user-images.githubusercontent.com/59229758/90457091-3eca2800-e0c0-11ea-9d28-29bfb64f48eb.png)

- [x] [REQUIRED] For the same footer section, the logos and the text below them should be links. Kindly fix it.

- [ ] [REQUIRED] The sections should have a container with a max-width property. Also, your responsive units for the text are not properly used. Please take a look at how the letters maintain a fixed size even on very large screens.

![image](https://user-images.githubusercontent.com/59229758/90457776-1e9b6880-e0c2-11ea-8dae-2afe7c73a0ac.png)


## Medium Screen

- [x] [RQUIRED] The footer size goes wild on medium size screen. The spacing between the links is also off. Please fix it. (This also happens on small screens).

![image](https://user-images.githubusercontent.com/59229758/90457194-8355c380-e0c0-11ea-880b-8f913e26e2ca.png)

## Small Screen

- [x] [REQUIRED] The navbar has a black element below the TNW logo. Please add it.

![image](https://user-images.githubusercontent.com/59229758/90457266-b0a27180-e0c0-11ea-931e-64b21bb5cf22.png)

- [x] [REQUIRED] The latest funding section should occupy the same width as the other sections. Please fix it.

![image](https://user-images.githubusercontent.com/59229758/90457327-d760a800-e0c0-11ea-84a3-d5d215bffdef.png)

- [x] [REQUIRED] All the images throughout the project should be links. Please add the appropriate `\a|` tag wrapping them.

## General Requirements.

- [x] [REQUIRED] Files not related to the project should not be committed and added to .gitignore (e.g. .DS_Store) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). In your case, in the assets folder you have a whole bunch of files unrelated to the project, even a  demo.html file which seems to be causing you some tremendous headaches with the linters. Please remove them.

- [ ] [REQUIRED] Please fix the lint errors. Once you get rid of all the files you don't need it will be much easier.

## Section specific.

- [ ] [REQUIRED] There are many repeated CSS rules throughout the project, for example lines 1388 through 1405, you have font-size: 0.5rem 3 times in a row! You even made a flex class but you repeated the display: flex rule almost 20 times! Please clean out your CSS and avoid repeating rules.

- [x] [OPTIONAL] Double-check your margin in the footer and make the footer logos smaller to make them look more like the original website. Your project is so good you should strive to make it perfect!

![image](https://user-images.githubusercontent.com/59229758/90457153-6b7e3f80-e0c0-11ea-96d9-afd20c7e82f5.png)

- [ ] [OPTIONAL] The categories section on small screens is a little off. The sizing is different and the margins between articles. The devil is in the details 😈 

- [ ] [OPTIONAL] The sizing of the main articles is off. Consider fixing it! 

![image](https://user-images.githubusercontent.com/59229758/90457671-cc5a4780-e0c1-11ea-8367-4669489766a0.png)

## That is it!

<img src="https://media.giphy.com/media/EDt1m8p5hqXG8/giphy.gif" />

It sounds like a lot, **I know** but it is mostly small changes, so you will be ready in no time for another review. 

Please implement the required changes, and consider implementing the optional ones. Please reach out if you have any questions.

It was a pleasure reviewing your code,

Happy coding! 💻

Slack: Luis Angel Maldoando
[LinkedIn](https://www.linkedin.com/in/lmaldonadoch/)

Blaise-Shyaka committed:
turn footer text into links

Blaise-Shyaka committed:
style the footer social media links

elasfarc committed:
fix latest funding section occupied predefined width

so now latest funding section occupies the same width as the other sections.

elasfarc committed:
adding sections wrapper with max-width

elasfarc committed:
.latest-funding .card (add padding between cards)

Blaise-Shyaka committed:
fix mobile version styles of the footer

Blaise-Shyaka committed:
refactor footer tablet view styles

Blaise-Shyaka committed:
improve footer styles on desktop

Blaise-Shyaka committed:
Merge branch 'TNW-homepage' of https://github.com/elasfarc/TheNextWeb-responsive-clone into TNW-homepage

elasfarc committed:
.latest-funding .top .ul (links after orange heading )

ipad / desktop : remove the right-border and add an ::after element for first and second li (ipad) but only second and last li (desktop)
desktop:
-translate ul to be on the same baseline of the orange heading (h4)
-change the flex order for li elements

Blaise-Shyaka committed:
add margin-top to the footer

elasfarc committed:
all images clickable

elasfarc committed:
remove not-used fonts

elasfarc committed:
adding .blue-icon at footer in html

elasfarc committed:
(nav)adding a black element below the TNW logo

Blaise-Shyaka committed:
add a fixed blue wheelchair

Blaise-Shyaka committed:
Merge branch 'TNW-homepage' of https://github.com/elasfarc/TheNextWeb-responsive-clone into TNW-homepage

Blaise-Shyaka committed:
fix linter errors

Blaise-Shyaka committed:
fix webhint error

MuriloRoque reviewed (approved):
# Status: Approved :green_circle: 

## Hello @Blaise-Shyaka and @elasfarc :wave: 

After all of those requests that my colleague @lmaldonadoch made, I must say that this is also my face after checking each requirement:

![giphy](https://media.tenor.com/images/4175442d21aa9ffa7baebe40739eefd4/tenor.gif)

You guys must have worked for TNW! I mean, you weren't supposed to make an **exact** clone :laughing: 

### Congratulations, you may merge, good luck with your next projects! :rocket: :clap: :clap: :clap: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[![Twitter Follow](https://img.shields.io/twitter/follow/MuriloRoquePai1?label=Follow%20Murilo%20on%20Twitter&style=social)](https://twitter.com/MuriloRoquePai1)
